hack/ccm/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG GOLANG_IMAGE=golang:1.17
+ARG TARGETPLATFORM=linux/amd64
+ARG ARCH=amd64
+
+# Build vpcctl binary
+FROM ${GOLANG_IMAGE} as vpc-builder
+ARG ARCH
+WORKDIR /build
+RUN git clone https://github.com/openshift/cloud-provider-vpc-controller
```

Comment:
checking out a specific commit will be a better idea than from master

Reply:
Updated it in the latest commit